app/buckets/handlers.js

```diff
@@ -42,7 +42,8 @@ exports.bucket_details = (req, res, next) => {
         bucket,
         apps_options: apps.exclude(bucket.apps),
         users_options: users.exclude(bucket.users),
-        current_user_is_bucket_admin
+        current_user_id: req.user.auth0_id,
```

Comment:
You don't need to pass this to the template, it's already available as `current_user.auth0_id`

Reply:
👍